src/app/components/homepage/carousel/carousel.component.css

```
@@ -0,0 +1,14 @@
+img{
+    height: 450px;
+    width: 992px;
+    margin-left: 60px;
+    border-radius: 10px;
+}
+
+.row{
+    position: absolute;
+    left: -127px;
+    top: 0px;
```

0 values should not have units
```suggestion
    top: 0;
```